tools/bumper-bot/parser/parser.test.go

```diff
@@ -0,0 +1,46 @@
+package parser
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = Describe("Testing internal parser functions", func() {
+	Describe("ParseComponentsYaml function", func() {
+		Context("When yaml file is empty", func() {
```

Comment:
lower case `w`.

so it would read:

`Testing internal parser ParseComponentsYaml function when yaml file is empty`